### PR TITLE
first page setup

### PR DESCRIPTION
**Pull request by sprakashkumar079**
this is first page setup where simply make home page
using scss and bootstrap

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="center jumbotron"> 
+  <h1>Welcome to world with Facebook</h1> 
+  <h2>
+    This is the home page for the
+    <a href="https://hidden-ocean-29030.herokuapp.com/">Facebook</a>
+    .
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]: remove space

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#appliction controller for project
 class ApplicationController < ActionController::Base
 	def hello
```
Comment (DeepakSen9983) [inline app/controllers/application_controller.rb]: make indentation and write method comments

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer"> 
+	<small>
```
Comment (DeepakSen9983) [inline app/views/layouts/_footer.html.erb]: follow indentation

Comment (DeepakSen9983) [inline app/views/layouts/_footer.html.erb]: follow indentation

Comment (DeepakSen9983) [inline app/views/layouts/_footer.html.erb]: follow indentation

Comment (DeepakSen9983) [inline app/views/layouts/_footer.html.erb]: follow indentation

**Review by DeepakSen9983 — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 #appliction controller for project
 class ApplicationController < ActionController::Base
+    #this is simple action call for first time
```
Comment (DeepakSen9983) [inline app/controllers/application_controller.rb]: please write uses of this action and also write routes
fix indentation issue

**Review by DeepakSen9983 — CHANGES_REQUESTED** [review]

**Review by DeepakSen9983 — APPROVED** [review]

File: app/assets/stylesheets/custom.scss [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+section {
+  overflow: auto; 
+}
+
+textarea {
+  resize: vertical; 
+}
+.center {
```
Comment (harsh-ind) [inline app/assets/stylesheets/custom.scss:15]: Please use proper indentation!

File: app/controllers/static_pages_controller.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+#home controller for first page 
+class StaticPagesController < ApplicationController
+  def home
+  end
```
Comment (harsh-ind) [inline app/controllers/static_pages_controller.rb]: No need to declare this method as it doesn't containing anything, You can just make a view file

File: app/views/layouts/_shim.html.erb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```
Comment (harsh-ind) [inline app/views/layouts/_shim.html.erb:4]: Can you explain why we are using this script

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+	<div class="container">
+       <%= link_to "FACEBOOK", root_path, id: "logo" %>
+       <nav> 
+	       <ul class="nav navbar-nav navbar-right"> 
+		        <li><%= link_to "Home", root_path %></li> 
+		 
+		        <li><%= link_to "Log in", '#' %></li>
+            </ul>
+        </nav>
+    </div>
```
Comment (harsh-ind) [inline app/views/layouts/_header.html.erb]: Please use proper indentation

File: app/views/layouts/_footer.html.erb [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+        The <a href="https://hidden-ocean-29030.herokuapp.com/">Facebook</a>
+        by <a href="https://github.com/sprakashkumar079/">Prakash</a>
+    </small> 
+    <nav> 
+    	<ul>
+    		<li><%= link_to "About", root_path %></li>
+    		<li><%= link_to "Contact", root_path %></li> 
+        </ul>
+    </nav>
+</footer>
```
Comment (harsh-ind) [inline app/views/layouts/_footer.html.erb:13]: Please use proper indentation

**Review by harsh-ind — CHANGES_REQUESTED** [review]

File: app/views/layouts/_shim.html.erb (hunk 7 above)
Comment (sprakashkumar079) [reply app/views/layouts/_shim.html.erb:4]: use for if the version of Microsoft Internet Explorer (IE)
is less than 9.
it’s actually a conditional comment supported by Internet Explorer
browsers for just this sort of situation.

File: app/assets/stylesheets/custom.scss [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+  font-weight: bold; 
+}
+
+#logo:hover {
+  color: yellow;
+  text-decoration: unset; 
+}
+
+
+
```
Comment (mitaligaur) [inline app/assets/stylesheets/custom.scss:67]: Remove additional blank lines

File: app/views/layouts/_header.html.erb [hunk 11]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (mitaligaur) [inline app/views/layouts/_header.html.erb:1]: Remove blank line

**Review by mitaligaur — CHANGES_REQUESTED** [review]

**Pull request merged**